packages/components/src/local/plan-turn-form/index.stories.tsx

```diff
@@ -58,6 +58,35 @@ export const Default: React.FC = () => <PlanTurnForm
     }
   }}
   icon={iconData}
+  plansSubmitted={false}
+/>
+
+export const TurnsPlanned: React.FC = () => <PlanTurnForm
+  turnPlanned={postback}
+  formHeader="Planning header"
+  formData={{
```

Comment:
The spec file uses a `formData` mock but this does not? Is the content different in the story? If so, why?

Reply:
Aah, it's so we can use `knobs` to vary the speed.

Reply:
That's fair enough. I think you'd still be able to do that without repeating code but without looking into it, I'm not 100% sure about that.